sensor_msgs/LaserScan scan
float64 distance